pagopa/io-app-design-system
# feat(pictogram): [IOBP-969] add empty wallet and meter limit pictogram

## Short description
This pull request adds two new pictograms to the `Pictogram` component and integrates them into the existing structure. 

## List of changes proposed in this pull request
- Generate `PictogramEmptyWallet` and `PictogramMeterLimit.svg` file and components

## How to test
- Open the design system test app
- Tap on `Pictograms`
- Verify if new components are rendered (check also the dark mode)
